src/serviceWorker.ts

```diff
@@ -11,13 +11,16 @@
 // opt-in, read https://bit.ly/CRA-PWA
 
 const isLocalhost = Boolean(
+  //https://github.com/amilajack/eslint-plugin-compat/issues/376
```

Comment:
Is it possible to put the link together with the disable?

Reply:
comment is now under `/* eslint-disable */`

Reply:
i just merged with master